app/editor/src/features/admin/topics/TopicList.tsx

```diff
@@ -59,40 +55,89 @@ export const TopicList: React.FC = () => {
     }
   }, [api, items.length, loading]);
 
+  React.useEffect(() => {
+    changeParentOverflow();
+    window.addEventListener('resize', updateMaxHeight);
+  }, []);
```

Comment:
You must release the listener or this will cause a memory leak

Reply:
I think this will go away, see below.

Reply:
Updated based on the demo feedbacks, thanks